### DISCUSSION TITLE
2021-12-17 Website Steering Committee

### DISCUSSION THREAD

**Discussion by lauggh** [worldiaday/website#51] [Meetings]
Scheduled for: 
2021-12-16 at 7am - 8am (PST), 4pm - 5pm (CET)

Agenda: 
- Provide one place to understand project, documentation dashboard, task status (provide vision and process how we are going to manage the project)
- Present the deliverable of content inventory and audit analysis
- Review target user needs, research plan, and support needed for this research

**Comment by GhadaKandil** [comment]
As far as I understood the agenda is:

- Present the deliverable of content inventory and analysis. (this is instead of the revision of RFP as per Andrea's suggestion)
- Provide one place to understand project, documentation dashboard, task status (e.g. provide vision and process how we are going to manage the project)

**Reply by lauggh** [reply]
Thanks Ghada. I have updated the agenda.
